static/css/style.scss

```diff
@@ -698,7 +698,7 @@ a.yellow-text:focus {
 
     background-size: contain;
 
-    height: 864px;
+//     height: 864px;
```

Comment:
Don't comment, just remove this line completely. Else looks good to me.

Reply:
alright, will do that.